pages/association/board.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Board of Directors'
+description: 'The board makes decisions regarding the Comunica Association'
+---
+
+The [Comunica Association](/association/) has a Board of Directors
+that makes decisions with respect to the Comunica Association.
+This page describes details on who are the members of this board, what it does, and how it works.
+
+## Members
+
+* [Ruben Taelman](https://www.rubensworks.net/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Code maintainer and [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
+* [Pieter Colpaert](https://pietercolpaert.be/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Management contributor
+
+## Goals
+
+The Board of Directors makes decisions concerning the following topics:
+
+* Determine long-term goals via the [roadmap](/roadmap/).
+* Suggest priorities of issues to the maintainers for short-term development via the [project boards](https://github.com/orgs/comunica/projects).
+* Coordinate future of the Comunica Association
+
+Furthermore, board meetings can be used to evaluate the maintenance and development of Comunica and its related dependencies,
+which includes development by externals via the [Bounty Program](/association/bounties/).
+
+## Becoming a Board Member
+
+There are two ways to become a Board Member:
+
+1. Become a financial contributor via [Open Collective](https://opencollective.com/comunica-association) of at least the Silver tier.
+2. Become a regular [contributor](/contribute/) in any other way, with a dedication of at least two hours per week.
+
+## Decision-making Process
+
+At least once every quarter, the board virtually meets for a board meeting.
```

Comment:
I’d propose at least once a year. We’ll see how it goes, but you’ll notice once every quarter will be quite a lot to assemble all board members.